CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Setup
```

Comment:
The current content covers the process of setup dev environment. But we also need to describe the dev process, contribution types, and channels. Please take [VS Code Contribution Guide](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md) as an example.

Reply:
Another area we need to cover is the design. This extension consists of TypeScript code and Java code. We need to explain what each part does and how they work.

Reply:
I would suggest that we agree on a table of contents first. Then you can fill in contents for each topic.